.eslintrc.js

```diff
@@ -1,18 +1,16 @@
 module.exports = {
-    env: {
-        browser: true,
-        es2021: true,
-        'jest/globals': true,
-    },
-    extends: ['airbnb-base'],
-    parserOptions: {
-        ecmaVersion: 12,
-        sourceType: 'module',
-    },
-    plugins: ['jest'],
-    rules: {
-        indent: ['error', 4],
-        'operator-linebreak': [2, 'after', { overrides: { '?': 'after', ':': 'after' } }],
-        linebreak: 0,
-    },
+  env: {
+    es6: true,
+    browser: true,
+    es2021: true,
+  },
```

Comment:
remove es6 and add node flag. Also, don't forget about `jest/globals`

Reply:
It fixed bro for another PR

Reply:
We install from npm

Reply:
Its npm package